test/integration/banListTest.ts

```diff
@@ -225,3 +227,63 @@ describe('Test: We will not be able to ban ourselves via ACL.', function () {
         assert.equal(acl.literalAclContent().deny.length, 3);
     })
 })
+
+
+describe.only('Test: ACL updates will batch when rules are added in succession.', function () {
+    it('Will batch ACL updates if we spam rules into a BanList', async function () {
+        this.timeout(180000)
+        const mjolnir = config.RUNTIME.client!
+        const serverName: string = new UserID(await mjolnir.getUserId()).domain
+        const moderator = await newTestUser({ name: { contains: "moderator" }});
+        moderator.joinRoom(this.mjolnir.managementRoomId);
+        const mjolnirId = await mjolnir.getUserId();
+        // How many rooms are we going to protect? Do we need to protect a few and verify all the ACL updates were batched in each?
```

Comment:
Is this a question for me?

Reply:
Oops, no that was me before i wrote the test